kgforge/specializations/models/rdf/rdf_model_service_from_directory.py

```diff
@@ -73,18 +72,31 @@ def _build_shapes_map(self) -> Dict:
                         ?shape a rdfs:Class
                     }
                 }
-            } ORDER BY ?type"""
+            } ORDER BY ?type
+        """
         res = self._graph.query(query)
-        return {row["type"]: row["shape"] for row in res}
 
+        class_being_shaped_id_to_shape_uri: Dict[str, URIRef] = {
```

Comment:
What class_being_shaped_id_to_shape_uri means ?

Reply:
I've renamed it to class_to_shape. But it was supposed to be:
key: id of the class that is the target of the shape
value: uri of the shape